jekyll/_cci2/orb-author.md

```diff
@@ -9,279 +9,289 @@ version:
-* When stored as a project environment variable or Contexts resource, an [API token]({{ site.baseurl }}/2.0/managing-api-tokens) generated by an org admin can help automate the orb publishing process, enabling _any_ org member to potentially publish a production orb by using the [CircleCI CLI]({{ site.baseurl }}/2.0/local-cli/) in a CircleCI job.
-* To use the orbs you create, you will need to enable the "Allow Uncertified Orbs" setting in the Security section of CircleCI's Organization Settings page for your organization (`https://circleci.com/[vcs]/organizations/[org-name]/settings#security`).
-* If you are creating an orb, you can use development versions to avoid having your orb publicly/permanently listed in CircleCI's orb registry before it is ready.
+## Orb Development Kit
 
-### The following high-level steps are needed to publish your first orb:
+The Orb Development Kit refers to a suite of tools working together to make production-level orb development simple, with automatic testing and deployment on CircleCI.
 
-1) Claim a namespace (assuming you don't yet have one). For example:
+### Getting Started
 
-`circleci namespace create sandbox github CircleCI-Public`
+To begin creating your new orb with the Orb Developer Kit, first create a new repository on [GitHub.com](https://github.com).
 
-In this example we are creating the `sandbox` namespace, which will be linked to the GitHub organization `CircleCI-Public`.
+Ensure the Organization on GitHub is the owner for the [namespace]({{site.baseurl}}/2.0/orb-concepts/#namespaces) for which you are developing your orb. If this is your own personal organization and namespace, you need not worry.
 
-**Note:** When creating a namespace via the CircleCI CLI, be sure to specify the VCS provider.
 
-2) Create the orb inside your namespace. This doesn't generate any content, but rather reserves the naming for when the orb is published. For example:
+#### 1) Create new [GitHub repository](https://github.com/new).
+{:.no_toc}
 
-`circleci orb create sandbox/hello-world`
+The name of your repository is not critical but we recommend something similar to "myProject-orb".
 
-3) Create the content of your orb in a file. You will generally do this in your code editor in a git repo made for your orb, but, for the sake of an example, let's assume a file in `/tmp/orb.yml` could be made with a bare-bones orb like:
+![Orb Registry]({{site.baseurl}}/assets/img/docs/new_orb_repo_gh.png)
 
-`echo 'version: "2.1"\ndescription: "a sample orb"' > /tmp/orb.yml`
+When complete, you will be brought to a page confirming your new repository and you should the generated git URL. Take note of the `HTTPS` version of your git URL, we will use it during out configuration.
 
-4) Validate that your code is a valid orb using the CLI. For example, using the path above you could use:
+![Orb Registry]({{site.baseurl}}/assets/img/docs/github_new_quick_setup.png)
 
-`circleci orb validate /tmp/orb.yml`
 
```

Comment:
I think adding a step in between here to direct people to get the new repo locally would be good?

Reply:
Just a step saying to clone I suppose?

Reply:
You do not clone it. The CLI handles the rest.